specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -2618,6 +2663,54 @@
         }
       }
     },
+    "CheckEligibilityDefinition": {
+      "type": "object",
+      "description": "The check eligibility Connector credentials definition",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "Connector kind (eg aws)"
```

Comment:
Possible to express this as an enum?

Reply:
In normal cases I agree, but here I prefer leaving this as a string to be consistent with all the remaining cloud connectors apis.